.gitignore

```diff
@@ -30,3 +30,18 @@
 *.exe
 *.out
 *.app
+
+# environment variables
+.env
+secrets.hpp
+
+# vsc
+.vscode
+
+# python
+__pycache__
+
+# misc
+test.ino
+local.py
+.car
```

Comment:
I can't leave a comment there, so I am commenting here: You probably should not version control `.pyc` files